etsin_finder/frontend/js/components/qvain/licenses/accessType.jsx

```diff
@@ -20,59 +20,69 @@ export class AccessType extends Component {
   promises = []
 
   static propTypes = {
-    Stores: PropTypes.object.isRequired
+    Stores: PropTypes.object.isRequired,
   }
 
   state = {
     options: {
       en: [],
-      fi: []
+      fi: [],
     },
-    accessTypeValidationError: null
+    accessTypeValidationError: null,
   }
 
   componentDidMount = () => {
-    this.promises.push(getReferenceData('access_type')
-      .then(res => {
-        const list = res.data.hits.hits;
-        const refsEn = list.map(ref => (
-          {
+    this.promises.push(
+      getReferenceData('access_type')
+        .then((res) => {
+          const list = res.data.hits.hits
+          const refsEn = list.map((ref) => ({
             value: ref._source.uri,
             label: ref._source.label.en,
-          }
-        ))
-        const refsFi = list.map(ref => (
-          {
+          }))
+          const refsFi = list.map((ref) => ({
             value: ref._source.uri,
             label: ref._source.label.fi,
+          }))
+
+          const user = this.props.Stores.Auth.user
+
+          if (!user.isUsingRems) {
+            // Removing one option from the list "Vaatii luvan hakemista Fairdata palvelusta".
+            // Assuming the option to be the last in the list.
+            // Other ways to do this is to check it from url or from description.
+            // Any of these ways is somewhat bad.
+            // It is recommended that this removal is done in the server not in the client.
```

Comment:
use the uri from `constants.js` for filtering

Reply:
fixed as suggested